src/main/java/com/google/codeu/data/UserMeal.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.codeu.data;
+
+import java.util.*;
```

Comment:
I would explicitly import the things that you need from `java.util`. I think you just want `UUID`? (you'll need to double check). This helps prevent hard to track down bugs where you accidentally reference a class you didn't realize that you imported.